LICENSE

```diff
@@ -187,7 +187,7 @@
      same "printed page" as the copyright notice for easier
      identification within third-party archives.
 
-  Copyright [yyyy] [name of copyright owner]
+  Copyright 2019 The Apache Foundation
```

Comment:
The license itself provides a template here. Should we edit it?

Reply:
Yup ;)